Rakefile

```diff
@@ -36,4 +36,22 @@ Rake::TestTask.new do |t|
   t.verbose = true
 end
 
-task default: [:test, :rubocop]
+task default: [:test, 'rubocop:auto_correct']
```

Comment:
I agree with @maurogeorge, the `rubocop:auto_correct` should not be part of default.